### ISSUE TITLE
Menu components accessibility

### ISSUE THREAD

**Issue by sunarya-thito** [sunarya-thito/shadcn_flutter#9]
Currently, menu components (menubar, dropdown menu, and context menu) does not have accessibility features such as:
* moving focus using keyboard
* proper focus traversal
